capture_service/android_application.cc

```diff
@@ -266,10 +268,10 @@ absl::Status AndroidApplication::GfxrSetup()
     RETURN_IF_ERROR(m_dev.Adb().Run(
     absl::StrFormat("shell settings put global gpu_debug_layer_app %s", m_package)));
 
-    std::string capture_file_location = kDeviceCapturePath + m_gfxr_capture_file_directory + "/" +
-                                        m_package + ".gfxr";
+    std::string gfxr_capture_directory = std::string(kDeviceCapturePath) + "/" +
+                                         m_gfxr_capture_file_directory;
```

Comment:
consider absl::StrCat, they claim that its more efficient

https://abseil.io/docs/cpp/guides/strings#abslstrcat-and-abslstrappend-for-string-concatenation

```suggestion
    std::string gfxr_capture_directory = absl::StrCat(kDeviceCapturePath, "/", m_gfxr_capture_file_directory);
```

Reply:
fixed